rskj-core/src/test/java/co/rsk/vm/VmDslTest.java

```diff
@@ -118,53 +125,189 @@ void testPush0() throws FileNotFoundException, DslProcessorException {
-        Assertions.assertNotNull(callTransactionReceiptNew);
+        assertNotNull(callTransactionReceiptNew);
         byte[] statusCallNew = callTransactionReceiptNew.getStatus();
-        Assertions.assertNotNull(statusCallNew);
+        assertNotNull(statusCallNew);
         Assertions.assertEquals(1, statusCallNew.length);
         Assertions.assertEquals(1, statusCallNew[0]);
 
         short newGas = ByteBuffer.wrap(callTransactionReceiptNew.getGasUsed()).getShort();
 
         Block block4 = world.getBlockByName("b04");
-        Assertions.assertNotNull(block4);
+        assertNotNull(block4);
         Assertions.assertEquals(1, block4.getTransactionsList().size());
 
 
         Transaction creationTransactionOld = world.getTransactionByName("txCreateOld");
-        Assertions.assertNotNull(creationTransactionOld);
+        assertNotNull(creationTransactionOld);
         TransactionReceipt creationTransactionReceiptOld = world.getTransactionReceiptByName("txCreateOld");
-        Assertions.assertNotNull(creationTransactionReceiptOld);
+        assertNotNull(creationTransactionReceiptOld);
         byte[] statusCreationOld = creationTransactionReceiptNew.getStatus();
-        Assertions.assertNotNull(statusCreationOld);
+        assertNotNull(statusCreationOld);
         Assertions.assertEquals(1, statusCreationOld.length);
         Assertions.assertEquals(1, statusCreationOld[0]);
 
         Transaction callTransactionOld = world.getTransactionByName("txCallOld");
-        Assertions.assertNotNull(callTransactionOld);
+        assertNotNull(callTransactionOld);
         TransactionReceipt callTransactionReceiptOld = world.getTransactionReceiptByName("txCallOld");
-        Assertions.assertNotNull(callTransactionReceiptOld);
+        assertNotNull(callTransactionReceiptOld);
         byte[] statusCallOld = callTransactionReceiptOld.getStatus();
-        Assertions.assertNotNull(statusCallOld);
+        assertNotNull(statusCallOld);
         Assertions.assertEquals(1, statusCallOld.length);
         Assertions.assertEquals(1, statusCallOld[0]);
 
         short oldGas = ByteBuffer.wrap(callTransactionReceiptOld.getGasUsed()).getShort();
-        Assertions.assertTrue(newGas < oldGas);
+        assertTrue(newGas < oldGas);
+    }
+
+    @Test
+    void testInitCodeSizeValidationSuccessWithoutInitcodeCostViaCreateOpcodeCREATE() throws FileNotFoundException, DslProcessorException {
+        TestSystemProperties rskip438Disabled = new TestSystemProperties(rawConfig ->
+                rawConfig.withValue("blockchain.config.hardforkActivationHeights.lovell700", ConfigValueFactory.fromAnyRef(-1))
+        );
+        DslParser parser = DslParser.fromResource("dsl/initcode_rskip438/create_opcode_test_without_initcode_cost_rskip_NOT_ACTIVE.txt");
+        World world = new World(rskip438Disabled);
+        WorldDslProcessor processor = new WorldDslProcessor(world);
```

Comment:
nice job on this! To clean things up a bit and avoid repeating code, how about adding a helper method to handle the setup for `DslParser`, `World`, and `WorldDslProcessor`?

Reply:
Ok, will do.